src/encoded/docs/schema-changes.md

```diff
@@ -154,44 +154,104 @@ This new object is an array of example objects that can successfully POST agains
 Updating an existing schema
 ----------------
 
-There are 2 categories when we talk about updating an existing schema:
+There are two situations we need to consider when updating an existing schema:
 
-* No affect on schema version
-* Update schema version
+### No change on schema version
 
+* For some schema changes, there will be no change to the schema version. These will be some minor changes to the schema. One such example includes adding a new enum to an existing list of enums. Another example will be adding a new property to existing schemas.
```

Comment:
Your guidelines are a little bit vague. I would go with something like:
Schema version is not to be updated if the change introduced is not going to cause a potential invalidation of existing objects in the database. For example an addition of a new enum value to a list of existing enums can not cause schema invalidation, but will simply extend the list of potential values to choose from.